src/m365/planner/commands/task/task-checklistitem-remove.ts

```diff
@@ -0,0 +1,124 @@
+import { Cli, Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+  taskId: string;
+  confirm?: boolean;
+}
+
+class PlannerTaskChecklistItemRemoveCommand extends GraphCommand {
+  public get name(): string {
+    return commands.TASK_CHECKLISTITEM_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Removes the checklist item from the planner task';
+  }
+
+  public defaultProperties(): string[] | undefined {
```

Comment:
Hi @akashkarda, we only use `defaultProperties` on list commands. Remove commands should not return anything.

Reply:
Thank you. I have implemented the changes. Can you please review?